test/core/event_engine/posix/event_poller_posix_test.cc

```diff
@@ -673,18 +677,20 @@ class Worker : public grpc_core::DualRefCounted<Worker> {
 
  private:
   void Work() {
-    auto result = g_event_poller->Work(24h, [this]() {
+    auto result = g_event_poller->Work(24h, [self = Ref()]() {
```

Comment:
This lgtm. I'm not sure if we could avoid the ref churn here, but at least now this guarantees a living object until all Workers are finished.

Reply:
Yes thats true